packages/react-dev-utils/openBrowser.js

```diff
@@ -88,6 +88,15 @@ function startBrowserProcess(browser, url, args) {
     }
   }
 
+  // In cases where no browser is set, the open library fails with an ENOENT.
+  // This means that on Windows 10 with Windows Subsystem For Linux, if the user
+  // has not set the Windows paths in their PATH, this will fail.
+  // Thus, if no browser is provided, we return early to avoid an (inevitable) crash.
+  // This solves https://github.com/facebook/create-react-app/issues/7251
+  if (browser === undefined) {
```

Comment:
I'm not sure this is the right fix. It may fix the issue specifically for WSL, but this addition is confusing with the following logic that sets the `browser = undefined`. Should the issue be solved from the `open` side?